azure/packages/azure-client/src/test/TestDataObject.ts

```diff
@@ -21,3 +23,64 @@ export class TestDataObject extends DataObject {
         super(props);
     }
 }
+
+export class TestDataObject2 extends DataObject {
+    private _counter: SharedCounter | undefined;
+
+    constructor(props: IDataObjectProps) {
+        super(props);
+    }
```

Comment:
Can probably just omit the constructor entirely if it's not doing anything here (same for TestDataObject above and BadTestDataObject below)

Reply:
yup - this file needed a cleanup. changed a context quite a bit while I was trying to repro DataObject bug we are chatting about on General.